products/jse/features/jse_full_html.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Ensure Site Owners can edit in full HTML
```

Comment:
We need to have editors be able to edit in full html too. It really messes things up if the site owner edits in HTML. The editor comes in and can't access that field.